UPGRADE-7.2.md

```diff
@@ -68,6 +68,8 @@ Notifier
    The `testIncompleteDsnException()` and `testMissingRequiredOptionException()` tests are no longer provided by default. If you make use of them (i.e. by implementing the
    `incompleteDsnProvider()` or `missingRequiredOptionProvider()` data providers), you now need to use the `IncompleteDsnTestTrait` or `MissingRequiredOptionTestTrait` respectively.
 
+ * Deprecate the `Sms77` transport, use `SevenIo`instead
```

Comment:
```suggestion
 * Deprecate the `Sms77` transport, use `SevenIo` instead
```

Reply:
I moved this with missing space in Upgrade-7.3